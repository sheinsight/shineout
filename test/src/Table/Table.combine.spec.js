import React from 'react'
import { Table } from 'shineout'
import Render from 'react-test-renderer'

const columns = [
  {
    title: 'id',
    render: 'id',
    width: 70,
  },
  { title: 'First Name', group: 'Name', render: 'firstName' },
  { title: 'Last Name', group: 'Name', render: 'lastName' },
  {
    title: 'Start Date',
    render: 'start',
    rowSpan: (a, b) => a.start === b.start,
    colSpan: d => {
      const hour = parseInt(d.time.slice(0, 2), 10)
      if (hour > 21 || hour < 9) return 2
      return 1
    },
  },
  { title: 'Time', render: 'time' },
  { title: 'Office', render: 'office5', rowSpan: true },
]

const data = [
  {
    id: 5839,
    firstName: 'Ryley',
    lastName: 'Ruecker',
    position: 'Marketing Designer',
    start: '2005-06-26',
    time: '12:30',
    salary: 329526,
    country: 'Lebanon',
    office: 'Mexico City',
    office5: 'Baghdad',
    height: 163.12,
  },
  {
    id: 6558,
    firstName: 'Patience',
    lastName: 'Stark',
    position: 'Secretary',
    start: '2005-06-26',
    time: '10:35',
    salary: 241073,
    country: 'Syrian Arab Republic',
    office: 'Atlanta',
    office5: 'Osaka-Kobe-Kyoto',
    height: 87.46,
  },
  {
    id: 2418,
    firstName: 'Myrtis',
    lastName: 'Denesik',
    position: 'Support Engineer',
    start: '2005-06-27',
    time: '03:46',
    salary: 239196,
    country: 'Bosnia and Herzegovina',
    office: 'Mexico City',
    office5: 'Baghdad',
    height: 130.66,
  },
  {
    id: 2612,
    firstName: 'Mollie',
    lastName: 'Kling',
    position: 'Javascript Developer',
    start: '2005-06-27',
    time: '10:38',
    salary: 131931,
    country: 'Ghana',
    office: 'Tokyo',
    office5: 'Fuzhou',
    height: 131.5,
  },
  {
    id: 6257,
    firstName: 'Cecilia',
    lastName: 'Wisozk',
    position: 'Support Lead',
    start: '2005-06-28',
    time: '01:04',
    salary: 159013,
    country: 'Falkland Islands (Malvinas)',
    office: 'London',
    office5: 'Berlin',
    height: 94.45,
  },
  {
    id: 6428,
    firstName: 'Lela',
    lastName: 'Murray',
    position: 'Accountant',
    start: '2005-06-28',
    time: '07:01',
    salary: 130936,
    country: 'Syrian Arab Republic',
    office: 'Mexico City',
    office5: 'Fuzhou',
    height: 133.12,
  },
  {
    id: 9469,
    firstName: 'Vernie',
    lastName: 'Ondricka',
    position: 'Integration Specialist',
    start: '2005-06-29',
    time: '09:44',
    salary: 267203,
    country: 'Wallis and Futuna',
    office: 'Chengdu',
    office5: 'Osaka-Kobe-Kyoto',
    height: 123.94,
  },
  {
    id: 7841,
    firstName: 'Maud',
    lastName: 'Gutkowski',
    position: 'Integration Specialist',
    start: '2005-06-30',
    time: '11:21',
    salary: 406431,
    country: 'Mexico',
    office: 'Chicago',
    office5: 'Baghdad',
    height: 149.17,
  },
  {
    id: 9634,
    firstName: 'Dixie',
    lastName: 'Strosin',
    position: 'Marketing Designer',
    start: '2005-06-30',
    time: '02:14',
    salary: 300638,
    country: 'Mexico',
    office: 'Delhi',
    office5: 'Osaka-Kobe-Kyoto',
    height: 145.6,
  },
  {
    id: 4758,
    firstName: 'Henderson',
    lastName: 'McCullough',
    position: 'Sales Assistant',
    start: '2005-07-01',
    time: '12:47',
    salary: 438465,
    country: 'Netherlands',
    office: 'Ho Chi Minh City',
    office5: 'Osaka-Kobe-Kyoto',
    height: 80.78,
  },
]
describe('Table[combine]', () => {
  test('should combine with pagination', () => {
    const wrapper = Render.create(
      <Table keygen="id" columns={columns} data={data} pagination={{ current: 1 }} />
    ).toJSON()
    expect(wrapper).toMatchSnapshot()
  })
  test('should combine without pagination', () => {
    const wrapper = Render.create(<Table keygen="id" columns={columns} data={data} />).toJSON()
    expect(wrapper).toMatchSnapshot()
  })
})
