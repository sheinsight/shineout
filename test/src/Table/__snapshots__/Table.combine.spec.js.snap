// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table[combine] should combine with pagination 1`] = `
Array [
  <div
    className="so-table so-table-default so-table-hover so-table-vertical-top"
    style={Object {}}
  >
    <div
      className="so-table-head so-table-simple-head"
      style={
        Object {
          "overflowY": "hidden",
        }
      }
    >
      <table
        className=""
        style={
          Object {
            "width": undefined,
          }
        }
      >
        <colgroup>
          <col
            style={
              Object {
                "width": 70,
              }
            }
          />
          <col
            style={
              Object {
                "width": undefined,
              }
            }
          />
          <col
            style={
              Object {
                "width": undefined,
              }
            }
          />
          <col
            style={
              Object {
                "width": undefined,
              }
            }
          />
          <col
            style={
              Object {
                "width": undefined,
              }
            }
          />
          <col
            style={
              Object {
                "width": undefined,
              }
            }
          />
        </colgroup>
        <thead>
          <tr>
            <th
              className=""
              rowSpan={2}
            >
              <div
                className=""
              >
                <span>
                  id
                </span>
              </div>
            </th>
            <th
              className="so-table-center so-table-condensed"
              colSpan={2}
            >
              Name
            </th>
            <th
              className=""
              rowSpan={2}
            >
              <div
                className=""
              >
                <span>
                  Start Date
                </span>
              </div>
            </th>
            <th
              className=""
              rowSpan={2}
            >
              <div
                className=""
              >
                <span>
                  Time
                </span>
              </div>
            </th>
            <th
              className=""
              rowSpan={2}
            >
              <div
                className=""
              >
                <span>
                  Office
                </span>
              </div>
            </th>
          </tr>
          <tr>
            <th
              className="so-table-condensed"
              rowSpan={1}
            >
              <div
                className=""
              >
                <span>
                  First Name
                </span>
              </div>
            </th>
            <th
              className="so-table-condensed"
              rowSpan={1}
            >
              <div
                className=""
              >
                <span>
                  Last Name
                </span>
              </div>
            </th>
          </tr>
        </thead>
      </table>
    </div>
    <div
      className="so-table-simple-body"
      onScroll={[Function]}
    >
      <table
        className=""
        style={
          Object {
            "width": undefined,
          }
        }
      >
        <colgroup>
          <col
            style={
              Object {
                "width": 70,
              }
            }
          />
          <col
            style={
              Object {
                "width": undefined,
              }
            }
          />
          <col
            style={
              Object {
                "width": undefined,
              }
            }
          />
          <col
            style={
              Object {
                "width": undefined,
              }
            }
          />
          <col
            style={
              Object {
                "width": undefined,
              }
            }
          />
          <col
            style={
              Object {
                "width": undefined,
              }
            }
          />
        </colgroup>
        <tbody>
          <tr
            className="so-table-normal"
            onClick={[Function]}
          >
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              5839
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              Ryley
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              Ruecker
            </td>
            <td
              className=""
              colSpan={1}
              rowSpan={2}
              style={Object {}}
            >
              2005-06-26
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              12:30
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              Baghdad
            </td>
          </tr>
          <tr
            className="so-table-normal"
            onClick={[Function]}
          >
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              6558
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              Patience
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              Stark
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              10:35
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              Osaka-Kobe-Kyoto
            </td>
          </tr>
          <tr
            className="so-table-normal"
            onClick={[Function]}
          >
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              2418
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              Myrtis
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              Denesik
            </td>
            <td
              className=""
              colSpan={2}
              style={Object {}}
            >
              2005-06-27
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              Baghdad
            </td>
          </tr>
          <tr
            className="so-table-normal"
            onClick={[Function]}
          >
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              2612
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              Mollie
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              Kling
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              2005-06-27
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              10:38
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              Fuzhou
            </td>
          </tr>
          <tr
            className="so-table-normal"
            onClick={[Function]}
          >
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              6257
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              Cecilia
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              Wisozk
            </td>
            <td
              className=""
              colSpan={2}
              rowSpan={2}
              style={Object {}}
            >
              2005-06-28
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              Berlin
            </td>
          </tr>
          <tr
            className="so-table-normal"
            onClick={[Function]}
          >
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              6428
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              Lela
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              Murray
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              Fuzhou
            </td>
          </tr>
          <tr
            className="so-table-normal"
            onClick={[Function]}
          >
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              9469
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              Vernie
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              Ondricka
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              2005-06-29
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              09:44
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              Osaka-Kobe-Kyoto
            </td>
          </tr>
          <tr
            className="so-table-normal"
            onClick={[Function]}
          >
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              7841
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              Maud
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              Gutkowski
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              2005-06-30
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              11:21
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              Baghdad
            </td>
          </tr>
          <tr
            className="so-table-normal"
            onClick={[Function]}
          >
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              9634
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              Dixie
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              Strosin
            </td>
            <td
              className=""
              colSpan={2}
              style={Object {}}
            >
              2005-06-30
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              Osaka-Kobe-Kyoto
            </td>
          </tr>
          <tr
            className="so-table-normal"
            onClick={[Function]}
          >
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              4758
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              Henderson
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              McCullough
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              2005-07-01
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              12:47
            </td>
            <td
              className=""
              colSpan={1}
              style={Object {}}
            >
              Osaka-Kobe-Kyoto
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>,
  <div
    className="so-pagination so-pagination-default"
    style={Object {}}
  >
    <div
      className="so-pagination-links so-pagination-section so-pagination-section-ltr"
    >
      <a
        className="so-pagination-item so-pagination-item-ltr so-pagination-arrow"
        disabled={true}
        onClick={[Function]}
      >
        <svg
          viewBox="0 0 1024 1024"
        >
          <path
            d="M304.905 561.68c-11.785 0-23.57-4.496-32.562-13.488-17.984-17.983-17.984-47.139 0-65.122l402.522-402.522c17.982-17.983 47.14-17.983 65.122 0 17.984 17.984 17.984 47.14 0 65.123l-402.521 402.521c-8.992 8.993-20.776 13.488-32.562 13.488z"
          />
          <path
            d="M707.426 964.201c-11.785 0-23.57-4.496-32.561-13.488l-402.523-402.522c-17.984-17.983-17.984-47.139 0-65.122 17.983-17.983 47.14-17.983 65.123 0l402.521 402.522c17.984 17.983 17.984 47.139 0 65.122-8.991 8.992-20.776 13.488-32.561 13.488z"
          />
        </svg>
      </a>
      <a
        className="so-pagination-item so-pagination-item-ltr so-pagination-current"
        disabled={true}
        onClick={[Function]}
      >
        1
      </a>
      <a
        className="so-pagination-item so-pagination-item-ltr so-pagination-arrow"
        disabled={true}
        onClick={[Function]}
      >
        <svg
          viewBox="0 0 1024 1024"
        >
          <path
            d="M728.76 561.68c-11.785 0-23.57-4.496-32.561-13.488l-402.523-402.521c-17.984-17.984-17.984-47.14 0-65.123 17.983-17.983 47.14-17.983 65.123 0l402.521 402.522c17.984 17.984 17.984 47.14 0 65.122-8.991 8.993-20.776 13.488-32.561 13.488z"
          />
          <path
            d="M326.238 964.202c-11.785 0-23.57-4.496-32.562-13.488-17.984-17.983-17.984-47.139 0-65.122l402.522-402.522c17.982-17.983 47.14-17.983 65.122 0 17.984 17.984 17.984 47.14 0 65.122l-402.521 402.522c-8.992 8.992-20.776 13.488-32.562 13.488z"
          />
        </svg>
      </a>
    </div>
  </div>,
]
`;

exports[`Table[combine] should combine without pagination 1`] = `
<div
  className="so-table so-table-default so-table-hover so-table-vertical-top"
  style={Object {}}
>
  <div
    className="so-table-head so-table-simple-head"
    style={
      Object {
        "overflowY": "hidden",
      }
    }
  >
    <table
      className=""
      style={
        Object {
          "width": undefined,
        }
      }
    >
      <colgroup>
        <col
          style={
            Object {
              "width": 70,
            }
          }
        />
        <col
          style={
            Object {
              "width": undefined,
            }
          }
        />
        <col
          style={
            Object {
              "width": undefined,
            }
          }
        />
        <col
          style={
            Object {
              "width": undefined,
            }
          }
        />
        <col
          style={
            Object {
              "width": undefined,
            }
          }
        />
        <col
          style={
            Object {
              "width": undefined,
            }
          }
        />
      </colgroup>
      <thead>
        <tr>
          <th
            className=""
            rowSpan={2}
          >
            <div
              className=""
            >
              <span>
                id
              </span>
            </div>
          </th>
          <th
            className="so-table-center so-table-condensed"
            colSpan={2}
          >
            Name
          </th>
          <th
            className=""
            rowSpan={2}
          >
            <div
              className=""
            >
              <span>
                Start Date
              </span>
            </div>
          </th>
          <th
            className=""
            rowSpan={2}
          >
            <div
              className=""
            >
              <span>
                Time
              </span>
            </div>
          </th>
          <th
            className=""
            rowSpan={2}
          >
            <div
              className=""
            >
              <span>
                Office
              </span>
            </div>
          </th>
        </tr>
        <tr>
          <th
            className="so-table-condensed"
            rowSpan={1}
          >
            <div
              className=""
            >
              <span>
                First Name
              </span>
            </div>
          </th>
          <th
            className="so-table-condensed"
            rowSpan={1}
          >
            <div
              className=""
            >
              <span>
                Last Name
              </span>
            </div>
          </th>
        </tr>
      </thead>
    </table>
  </div>
  <div
    className="so-table-simple-body"
    onScroll={[Function]}
  >
    <table
      className=""
      style={
        Object {
          "width": undefined,
        }
      }
    >
      <colgroup>
        <col
          style={
            Object {
              "width": 70,
            }
          }
        />
        <col
          style={
            Object {
              "width": undefined,
            }
          }
        />
        <col
          style={
            Object {
              "width": undefined,
            }
          }
        />
        <col
          style={
            Object {
              "width": undefined,
            }
          }
        />
        <col
          style={
            Object {
              "width": undefined,
            }
          }
        />
        <col
          style={
            Object {
              "width": undefined,
            }
          }
        />
      </colgroup>
      <tbody>
        <tr
          className="so-table-normal"
          onClick={[Function]}
        >
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            5839
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            Ryley
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            Ruecker
          </td>
          <td
            className=""
            colSpan={1}
            rowSpan={2}
            style={Object {}}
          >
            2005-06-26
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            12:30
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            Baghdad
          </td>
        </tr>
        <tr
          className="so-table-normal"
          onClick={[Function]}
        >
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            6558
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            Patience
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            Stark
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            10:35
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            Osaka-Kobe-Kyoto
          </td>
        </tr>
        <tr
          className="so-table-normal"
          onClick={[Function]}
        >
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            2418
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            Myrtis
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            Denesik
          </td>
          <td
            className=""
            colSpan={2}
            style={Object {}}
          >
            2005-06-27
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            Baghdad
          </td>
        </tr>
        <tr
          className="so-table-normal"
          onClick={[Function]}
        >
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            2612
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            Mollie
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            Kling
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            2005-06-27
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            10:38
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            Fuzhou
          </td>
        </tr>
        <tr
          className="so-table-normal"
          onClick={[Function]}
        >
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            6257
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            Cecilia
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            Wisozk
          </td>
          <td
            className=""
            colSpan={2}
            rowSpan={2}
            style={Object {}}
          >
            2005-06-28
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            Berlin
          </td>
        </tr>
        <tr
          className="so-table-normal"
          onClick={[Function]}
        >
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            6428
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            Lela
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            Murray
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            Fuzhou
          </td>
        </tr>
        <tr
          className="so-table-normal"
          onClick={[Function]}
        >
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            9469
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            Vernie
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            Ondricka
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            2005-06-29
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            09:44
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            Osaka-Kobe-Kyoto
          </td>
        </tr>
        <tr
          className="so-table-normal"
          onClick={[Function]}
        >
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            7841
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            Maud
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            Gutkowski
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            2005-06-30
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            11:21
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            Baghdad
          </td>
        </tr>
        <tr
          className="so-table-normal"
          onClick={[Function]}
        >
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            9634
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            Dixie
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            Strosin
          </td>
          <td
            className=""
            colSpan={2}
            style={Object {}}
          >
            2005-06-30
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            Osaka-Kobe-Kyoto
          </td>
        </tr>
        <tr
          className="so-table-normal"
          onClick={[Function]}
        >
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            4758
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            Henderson
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            McCullough
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            2005-07-01
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            12:47
          </td>
          <td
            className=""
            colSpan={1}
            style={Object {}}
          >
            Osaka-Kobe-Kyoto
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
`;
